tide-whisperer.go

```diff
@@ -296,6 +296,16 @@ func main() {
 			log.Printf("Calling GetDiabeloopParametersHistory")
 			defaultLevelFilter := make([]int, 1)
 			defaultLevelFilter = append(defaultLevelFilter, 1)
+
+			var device string
+			var deviceErr error
+			if device, deviceErr = storage.GetDeviceModel(queryParams.UserId); deviceErr != nil {
+				log.Printf("Error in GetDeviceModel for user %s. Error: %s", queryParams.UserId, deviceErr)
+			}
+			if device == "DBLHU" {
+				defaultLevelFilter = append(defaultLevelFilter, 3)
```

Comment:
why is defaultLevelFilter set to 3? 
shouldn't be 2?

Reply:
So called "Level 2 parameters" are stored with a level 3 value...